packages/components/src/Popover/Popover.test.tsx

```diff
@@ -44,6 +45,31 @@ describe('Popover', () => {
     }
   })
 
+  test('Accessibility', () => {
+    renderWithTheme(
+      <Popover
+        id="a11y"
+        content={
+          <PopoverLayout header="Header text" footer>
+            content
+          </PopoverLayout>
+        }
+      >
+        <Button>Open</Button>
+      </Popover>
+    )
+
+    const openPopover = screen.getByText('Open')
+    fireEvent.click(openPopover)
+
+    expect(screen.queryByText('Header text')).toBeInTheDocument()
+    expect(screen.getByRole('dialog')).toBeInTheDocument()
+    expect(screen.getByRole('dialog')).toHaveAttribute(
+      'aria-labelledby',
+      'a11y-heading'
+    )
```

Comment:
We want to verify that the id specified in `aria-labelledby` exists.

```suggestion
    )
    expect(screen.getByText('Header text')).toHaveAttribute('id', 'a11y-heading')
```